spec/system/teacher_interface/application_spec.rb

```diff
@@ -865,18 +865,17 @@ def then_i_see_the_registration_number_form
     expect(registration_number_form.heading.text).to eq(
       "What is your registration number?",
     )
-
-    registration_number_form.details.summary.click
-    expect(registration_number_form).to have_content(
-      "Other teaching authority information.",
-    )
+    expect(registration_number_form).to have_content("Status information")
+    expect(registration_number_form).to have_content("Sanction information")
   end
 
   def then_i_see_the_upload_written_statement_form
     expect(upload_document_page).to have_title("Upload a document")
     expect(upload_document_page.heading.text).to eq(
       "Upload your written statement",
     )
+    expect(registration_number_form).to have_content("Status information")
+    expect(registration_number_form).to have_content("Sanction information")
```

Comment:
This is referring to a `registration_number_form` but in this test we would be on the `upload_document_page`.